spec-0007/index.md

```diff
@@ -97,44 +97,69 @@ The [deprecation strategy](https://github.com/scientific-python/specs/pull/180#i
 
    ```
 
-   Also note the suggested language for the `rng` parameter docstring.
+   Also note the suggested language for the `rng` parameter docstring, which encourages the user to pass a `Generator` or None, but allows for other types accepted by `numpy.random.default_rng`.
 
 ### Impact
 
-The following users will be affected:
-
-1. Those who use `np.random.seed`.
-   The proposal will do away with that global seeding mechanism, meaning that code that relies on it will, after a certain deprecation period, start seeing a different stream of random numbers than before.
-   To ensure that this does not go unnoticed, the library should raise a `FutureWarning` if `np.random.seed` was called earlier (we show how to do that further down).
+There are three classes of users, which will be affected to varying degrees.
 
-   Code will, in effect, go from being seeded to being unseeded. To avoid that from happening, users should switch from using `np.random.seed` to specifying the `rng` argument to all functions that allow it.
+1. Those who do not attempt to control the random state.
+   Their code will immediately switch from using the unseeded global `RandomState` to using an unseeded `Generator`.
+   Since the underlying _distributions_ of pseudo-random numbers will not change, these users will be relatively unaffected.
 
-2. Those who do not seed. Their code will, after the deprecation period, use the newly proposed default. Since they were already not requesting repeatable sequences, and since the underlying _distributions_ of pseudo-random numbers did not change, they should be unaffected.
+2. Users of `random_state`/`seed` arguments. 
+   Support for these arguments will be dropped eventually, but during the deprecation period, we can provide clear guidance, via warnings and documentation, on how to migrate to the new `rng` keyword.
 
-3. Users of `random_state=...`.
-   Support for the `random_state` argument may be dropped eventually, at which point using that keyword will raise an error.
-   Meanwhile we can provide clear guidance, via deprecation warnings and documentation, on how to migrate to the new `rng` keyword.
+3. Those who use `numpy.random.seed`. 
+   The proposal will do away with that global seeding mechanism, meaning that code that relies on it will, after the deprecation period, go from being seeded to being unseeded.
```

Comment:
```suggestion
   The proposal will do away with that global seeding mechanism, meaning that code that relies on it would, after the deprecation period, go from being seeded to being unseeded.
```